bindings/gumjs/runtime/hexdump.js

```diff
@@ -19,6 +20,7 @@ function hexdump(target, options) {
     if (length === undefined)
       length = 256;
     buffer = Memory.readByteArray(target, length);
+    startAddress = target.toInt32() >>> 0;
```

Comment:
This will truncate some pointers in 64-bit processes. In general never turn a pointer into a JavaScript number like here, because those are backed by `double`, which means your pointer might get rounded to a different address (lol).